ruby/open-location-code.gemspec

```diff
@@ -5,7 +5,7 @@ require "date"
 
 Gem::Specification.new do |s|
   s.name          = "open-location-code"
-  s.version       = "0.0.1"
+  s.version       = "0.1.0"
```

Comment:
How about we match the version with the git tag so that people can easily know what version they're getting? So that would be [1.0.2](https://github.com/google/open-location-code/tree/1.0.2).

Reply:
Thank you for your review. I update the commit and rename the branch. And I created a new pull request from new branch. https://github.com/google/open-location-code/pull/115

Could you close this PR and review new PR #115 ?